src/components/Image/Image.docs.mdx

```diff
@@ -0,0 +1,42 @@
+import { Story, Preview, Props } from '@storybook/addon-docs/blocks';
+import Image from './Image';
+
+# Image
+
+Displays an image. The `Image` component requires `height` and `width` to set aspect ratio and `sizes` and `srcSet` to provide [responsive rendering](https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images). They will not be calculated for you and there are no default values. Future components may provide a simpler implementation for specific use cases.
+
+This component does not provide [art direction](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/picture), which is also a task for a future component.
+
+## Usage
+
+```js
+import { Image } from '@quartz/interface/src/components';
+```
+
+## Props
+
+<Props of={Image} />
+
+## Usage guidelines
+
+- **Do** use a container to constrain the size or shape of the image. An `Image` will always expand or contract to fit the width of its container. The `Image` component purposefully does not accept a `className` prop.
+- **Do** understand that the `height` and `width` props are [used to help the browser reserve space](https://blog.logrocket.com/jank-free-page-loading-with-media-aspect-ratios/) for the image. They do not need to match the _rendered_ size of the image but should be set at reasonable upper bounds.
```

Comment:
Perhaps an aspectRatio prop would be more intuitive? If I didn't read this I'd expect width and height to enforce fixed dimensions since that's the behavior of the native img element

Reply:
I'll have to think about it. If we only required an aspect ratio, we'd have to guess at reasonable values of `height` and `width`. As that article notes, they have separate, important meaning:

> Any appropriate img width and height can be used since the image is resized by CSS (e.g., width="4" height="3"). However, it is best to set a reasonable size to ensure the image remains visible in very old browsers, if CSS fails to load, or when a user style sheet is defined.

We could instead require `aspectRatio` and a `maxWidth` prop. But there might be situations where `height` is the constraining dimension. I'd lean towards making this base component as agnostic as possible.

Reply:
Pivoting to `fallback` props per our conversation.